.gitignore

```diff
@@ -37,4 +37,4 @@ jspm_packages
 .node_repl_history
 
 # Build directory
-dist
+#dist
```

Comment:
Thanks for the PR @ocha. Yarn is just a dependency manager, it pulls dependencies from package repos just like NPM so you don't need to include the built files in the git repo, so you can drop off this commit.